From: Coly Li <bosong.ly@taobao.com>
Subject: fix typo of do_hypervisor_callback in entry_64.S
Date: Fri Dec  9 14:20:29 CST 2011
Patch-mainline: in-house
Reference:

This patch fixed a compiling error which is newly found
in RHEL 2.6.32-220 kernel.

Signed-off-by: Coly Li <bosong.ly@taobao.com>
---
Index: linux-2.6.32-220.0.0.el5/arch/x86/kernel/entry_64.S
===================================================================
--- linux-2.6.32-220.0.0.el5.orig/arch/x86/kernel/entry_64.S
+++ linux-2.6.32-220.0.0.el5/arch/x86/kernel/entry_64.S
@@ -1303,7 +1303,7 @@ ENTRY(xen_do_hypervisor_callback)   # do
 	decl PER_CPU_VAR(irq_count)
 	jmp  error_exit
 	CFI_ENDPROC
-END(do_hypervisor_callback)
+END(xen_do_hypervisor_callback)
 
 /*
  * Hypervisor uses this for application faults while it executes.
